cmiles/generator.py

```diff
@@ -101,9 +110,20 @@ def to_canonical_smiles(molecule, canonicalization='openeye'):
     else:
         raise TypeError("canonicalization must be either 'openeye' or 'rdkit'")
 
+    smiles['Standard_InChI'] = to_inchi(molecule)
```

Comment:
Great, glad this was easy. Do they have `InChIKey` available as well or is that a straight SHA hash of `InChI`.

Reply:
I can get it from RDKit and it's a good idea to do that. RDKit gets it from the InChI API. It's not a straight SHA hash even though it uses SHA hash function for the first 2 blocks. It seems like the `/p` segment is not hashed at all. 
https://www.inchi-trust.org/technical-faq/#13